content/en/docs/instrumentation/net/automatic/_index.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Automatic Instrumentation
+description: Send traces and metrics from .NET applications and services.
+linkTitle: Automatic
+cSpell:ignore: coreutils HKLM iisreset myapp
+weight: 20
+---
+
+Use the OpenTelemetry .NET Automatic Instrumentation to send traces and metrics
+from .NET applications and services to observability back ends without having to
```

Comment:
in other palces I found

```suggestion
from .NET applications and services to observability back-ends without having to
```

or

```suggestion
from .NET applications and services to observability backends without having to
```

Reply:
Following the Splunk style guide here ;) https://docs.splunk.com/Documentation/StyleGuide/current/StyleGuide/Usagedictionary#B